hrjobcontract/js/test/controllers/contract_test.js

```diff
@@ -1,4 +1,3 @@
-/* global inject */
```

Comment:
if I remove that line semistandard throws an error. For some reason inject is not declared as a global.